.github/workflows/jest-unit-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Jest unit tests
+on: push
+
+jobs:
+  jest-unit-tests:
+    runs-on: [ARM64, self-hosted, Linux]
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Install node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: '.node-version'
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libcurl4-openssl-dev
+      - name: Install package dependencies
+        run: |
+          npm ci --verbose --no-optional
+      - name: Run tests
+        run: |
+          npm test
```

Comment:
I tried to get these tests running with docker, but the tests are failing, and I couldn't get a clear indication as to what was causing the failure.  So I'm just going to leave it as running on the system.  

Since these are unit tests where a lot of data is mocked, I think this is acceptable, but if we want to pursue this in the future, here are the commands that have to be added before running tests

```
      - name: Create Docker network
        run: |
          docker swarm init
          npmdocker network create czidnet --attachable --driver overlay
      - name: Build Docker container for graphql mesh
        run: |
          make build
```